src/style/animatableManager.js

```diff
@@ -0,0 +1,122 @@
+import _ from 'lodash';
+
+
+/** Definitions */
+const definitions = {
+  basicListEntrance: {
+    from: {opacity: 0, translateY: 20},
+    to: {opacity: 1, translateY: 0},
+  }, 
+  cardEntrance: {
+    from: {opacity: 0, translateY: 40},
+    to: {opacity: 1, translateY: 0},
+  },
+};
+
+class AnimatableManager {
+  loadCustomDefinitions(costumDefinitions) {
+    return {
+      ...definitions,
+      ...costumDefinitions,
+    };
+  }
+
+  /** Presets */
+  // Commons
+  getFadeIn() {
+    return {
+      animation: 'fadeIn',
+      duration: 300,
+      easing: 'ease-out-quint',
+      useNativeDriver: true,
+    };
+  }
+
+  // Components
+  getDialogAnimationProps(options) {
```

Comment:
- I think the suffix of "AnimationProps" at the end of each method is redundant since it's clear we're talking about animaton presets, also it makes the names too long.

Reply:
Done.